Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1302,11 +1302,13 @@ def __init__(self, howMuch, dataset):
     ('WpM_13TeV_TuneCUETP8M1_cfi', UpgradeFragment(Kby(9,50),'WpM_13')),
     ('SingleNuE10_cfi', UpgradeFragment(Kby(9,50),'NuGun')),
     ('TTbarLepton_13TeV_TuneCUETP8M1_cfi', UpgradeFragment(Kby(9,50),'TTbarLepton_13')),
+    ('TTbarLepton_14TeV_TuneCUETP8M1_cfi', UpgradeFragment(Kby(9,50),'TTbarLepton_14')),
```

Comment:
I think we prefer to put new fragments at the end of the list to avoid changing all the workflow numbers, which could cause confusion and mistakes. @chayanit can you confirm?

Reply:
Right. This will ruin the order of existing workflows. Please add new workflows at the end of the list instead.